src/main/scala/akka/io/AsyncDnsResolver.scala

```diff
@@ -154,4 +183,26 @@ object AsyncDnsResolver {
 
   private[io] def isSrv(name: String) =
     name.nonEmpty && name(0) == '_'
+
+  private[io] val NameserverLine = """^\s*nameserver\s+(.*)$""".r
+
+  def parseSystemNameServers(resolvConf: File): Option[immutable.Seq[InetSocketAddress]] =
+    try {
+      val addresses =
+        for {
+          line <- Source.fromFile(resolvConf).getLines()
+          addr <- NameserverLine.findFirstMatchIn(line).map(_.group(1))
+        } yield parseNameserverAddress(addr)
+      Some(addresses.toList)
+    } catch {
+      case NonFatal(_) => Option.empty
```

Comment:
@huntc CAn't we give a more precise waring if nameservers are empty and resolve-conf is off?